extras/po/fr.po

```diff
@@ -882,7 +882,7 @@ msgstr ""
 #: tomb:Open:mount_tomb:1830
 msgid "Last visit by ::1 user::(::2 tomb build::) from ::3 tty:: on ::4 host::"
 msgstr ""
-"Dernière visite par ::1 user::(::2 tomb build) depuis ::3 tty:: sur ::4 "
+"Dernière visite par ::1 user::(::2 tomb build::) depuis ::3 tty:: sur ::4 "
 "host::"
```

Comment:
Oh yes, you are right.

As you can see, I just updated the second line, it was already split. I guess it's because of the 80 characters limit.